conda/fetch.py

```diff
@@ -145,6 +145,11 @@ def fetch_repodata(url, cache_dir=None, use_cache=False, session=None):
         log.debug(msg)
         raise RuntimeError(msg)
 
+    except requests.exceptions.SSLError as e:
```

Comment:
SSLError subclasses from ConnectionError, meaning that if somehow a 407 gives this exception the https proxy handling will break.